.github/workflows/mac-mps.yml

```diff
@@ -0,0 +1,35 @@
+name: Mac MPS
+
+on:
+  push:
+    tags:
+      - ciflow/mps/*
```

Comment:
You might need to update probot to recognize this in [handleLabelEvent](https://github.com/pytorch/test-infra/blob/4bf7a03310e7b9085291727e44ae22135d6fc043/torchci/lib/bot/ciflowPushTrigger.ts#L137). May be you're already on that

Reply:
Yep, created https://github.com/pytorch/test-infra/pull/524